resources/lib/services/cache/cache_management.py

```diff
@@ -276,7 +276,7 @@ def delete_expired(self):
             bucket_content = self._get_cache_bucket(bucket['name'])
             for identifier, cache_entry in list(bucket_content.items()):  # pylint: disable=unused-variable
```

Comment:
can you remove also the pylist disable? i think is no more needed

Reply:
You can do this yourself, edit the branch in GitHub, or using git CLI before merging.
There is no need to ask for this, you are the author and I opened up my branch to you.
Especially since making the change is faster than asking ;-)